packages/webapp/src/_app/ui/expander.tsx

```diff
@@ -5,27 +5,30 @@ import { Container } from "_app";
 
 interface ExpanderProps {
     header: React.ReactNode;
-    hideContentDivider?: boolean;
+    showContentDivider?: boolean;
     startExpanded?: boolean;
+    inactive?: boolean;
     children: React.ReactNode;
 }
 
 export const Expander = ({
     header,
-    hideContentDivider,
+    showContentDivider,
```

Comment:
Does we use a particular form of this enough to provide a default value?

Reply:
It defaults to `undefined` right now, which is falsy. BUT...I think adding a default value in does make it more explicit...easier to see what's actually going on. Added.